src/hooks/useEvent.ts

```
@@ -0,0 +1,39 @@
+};
+
+export const useEventMutation = () => {
+  const validationMutation = useSWRMutation<
+    ParticipantValidationResponse,
+    AxiosError,
+    string,
+    ParticipantValidationDto
+  >("/participant/event-participations/validate-applicable", eventUpdater);
+
+  const submitEventMutation = useSWRMutation<
+    any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)